camel/models/base_model.py

```diff
@@ -0,0 +1,37 @@
+# =========== Copyright 2023 @ CAMEL-AI.org. All Rights Reserved. ===========
+# Licensed under the Apache License, Version 2.0 (the “License”);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an “AS IS” BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =========== Copyright 2023 @ CAMEL-AI.org. All Rights Reserved. ===========
+from abc import ABC, abstractmethod
+from typing import Any, Dict, List
+
+
+class BaseModel(ABC):
```

Comment:
Should we call it `BaseModelBackend`? We may use pydantic in the future to enforce type hints.  Pydantic has the same class named `BaseModel`.

Reply:
See. You may want to have the same name for two classes if they are in different places. This is the reason not to pile stuff into __init__.py. Let's rename, no problem.